tests/python_client/testcases/test_connection.py

```diff
@@ -542,7 +542,7 @@ def test_connection_connect_params(self, host, port, connect_name):
                                                  check_items={ct.dict_content: {"address": f"{host}:{port}",
                                                                                 "user": ""}})
 
-    @pytest.mark.tags(ct.CaseLabel.L1)
+    @pytest.mark.tags(ct.CaseLabel.L2)
```

Comment:
Curious about why change the test case Level?

Reply:
@yanliang567  aggreed. 
You CAN NOT pass  this case in your local machine！！
I usually run the L0 L1 tests locally before submitting the pr. 
so...